pkg/aws/client_test.go

```diff
@@ -516,7 +516,7 @@ var _ = Describe("Client", func() {
 			valid, err := client.ValidateCredentials()
 			Expect(valid).To(BeFalse())
 			Expect(err.Error()).To(ContainSubstring(
-				"Invalid AWS Credentials: %s.\n For help configuring your credentials, see", errMsg))
+				"invalid AWS credentials: InvalidClientTokenId: bad credentials.\n For help configuring your credentials, see"))
```

Comment:
Why was this error message changed so much? The linter error was only for `Invalid` being capitalized wasn't it?